onset_maproom/maproom.py

```diff
@@ -435,6 +581,7 @@ def cess_plots(
 # TODO can we make this any more redundant?
 @APP.callback(
     Output("onset_layer", "url"),
+    Input("yearly_stats_input", "value"),
```

Comment:
This callback has 11 `Input`s, but this is the only one whose name ends in `_input`. I would remove that.

Also, as I commented elsewhere, "yearly" doesn't seem appropriate now that "monit" is an option.